spec/openapi.yaml

```diff
@@ -916,11 +648,6 @@ components:
             origin:
               enum: ["local", "sdcard"]
```

Comment:
How about change sdcard to USB? Not sure if the apps can handle it.
 origin: ["local", "usb"]

Reply:
I havent tried. But I think there is no point for the app to support it. I wold keep sdcard for sake of compatibility.